packages/p2p/src/components/toggle-container/toggle-container.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type TToggleContainer = {
+    children: React.ReactNode;
+};
+
+const ToggleContainer = ({ children }: TToggleContainer) => (
```

Comment:
```suggestion
const ToggleContainer: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => (
```
@farhan-nurzi-deriv It's better to use React generics, So if they change something we don't have to.
No need to make the change tho 😅 Please ignore this comment 🙇🏻

Reply:
Just update the code haha, i did the the same thing ☝️

Reply:
@farhan-nurzi-deriv Damn too fast :)))